lib/logstash/inputs/twitter.rb

```diff
@@ -121,4 +121,32 @@ def run(queue)
       retry
     end
   end # def run
+
+  protected
+  def process_a_tweet( tweet, queue)
+    if tweet.is_a?(Twitter::Tweet)
```

Comment:
In that case, let's remove an indentation level by turning the first conditional into an early return:

```ruby
def process_a_tweet(tweet, queue)
  return unless tweet.is_a?(Twitter:Tweet)
  ...
```